ui/imports/shared/popups/send/Helpers.qml

```diff
@@ -0,0 +1,69 @@
+pragma Singleton
+
+import QtQuick 2.15
+import QtQml 2.15
+
+import utils 1.0
+import shared.stores 1.0
+import shared.stores.send 1.0
+import AppLayouts.Wallet.stores 1.0 as WalletStores
+
+import StatusQ.Core 0.1
+
+import "./panels"
+import "./controls"
+import "./views"
+
+QtObject {
+    id: root
+
+    function createSendModalRequirements() {
+        return {
+            preSelectedAccount: null,
+            preSelectedRecipientType: TabAddressSelectorView.Type.Address,
+            preSelectedRecipient: null,
+            preSelectedHoldingType: 0,
+            preSelectedHolding: null,
+            preSelectedHoldingID: "",
+            preDefinedAmountToSend: "",
+            preSelectedSendType: Constants.SendType.Transfer
+        }
+    }
+
+    function lookupAddressesForSendModal(senderAddress, recipientAddress, asset, isCollectible, amount) {
```

Comment:
```suggestion
    function lookupAddressesForSendModal(senderAddress, recipientAddress, token, isCollectible, amount) {
```

Reply:
Just a naming suggestion since in the UI context generally it is used `asset` to define `ERC20` tokens and `collectible` for `ERC721`, `token` as a general term.

Reply:
I changed it. Still, my understanding is the other way that asset is the general term for any kind of contract-based token or native currency that this variable can hold.

Reply:
Maybe yes in general terms but in the UI context that's the differentiation we have right now! Thanks for the update!